lib/reek/examiner.rb

```diff
@@ -25,8 +25,8 @@ class Examiner
     #   The configuration for this Examiner.
     #
     # @public
-    def initialize(source,
-                   filter_by_smells = [],
+    def initialize(source:,
+                   filter_by_smells: [],
```

Comment:
I'm not sure how I feel about making the main argument a keyword argument as well. It seems a little superfluous. Any reasons making all arguments keyword arguments would be better?